tsl/test/expected/cagg_api.out

```diff
@@ -0,0 +1,159 @@
+SET search_path TO _timescaledb_functions, public;
+-- These are not part of the API, but we test them here just to make
+-- sure they work as expected.
+SELECT table_name, get_materialization_info(table_name)
+  FROM (
+      VALUES ('tstz_temperature_15m'), ('multi_temperature_15m')
+  ) t(table_name);
+      table_name       | get_materialization_info 
+-----------------------+--------------------------
+ tstz_temperature_15m  | (8,"@ 15 mins")
+ multi_temperature_15m | (9,"@ 15 mins")
+(2 rows)
+
+\set ON_ERROR_STOP 0
+SELECT get_materialization_info('hyper_no_cagg');
+ERROR:  "public.hyper_no_cagg" is not a continuous aggregate
+\set ON_ERROR_STOP 1
+SELECT * INTO before FROM _timescaledb_catalog.continuous_aggs_materialization_invalidation_log;
+CALL _timescaledb_functions.add_materialization_invalidations(
+     'tstz_temperature_15m'::regclass,
+     '["2025-04-25 11:10:00+02","2025-04-26 11:14:00+02"]'::tsrange
+);
+SELECT continuous_aggregate,
+       to_timestamp(lhs.lowest_modified_value),
+       to_timestamp(lhs.greatest_modified_value)
+  FROM _timescaledb_catalog.continuous_aggs_materialization_invalidation_log lhs
+  JOIN continuous_aggregates USING (materialization_id)
+LEFT JOIN before rhs ON row(lhs.*) = row(rhs.*)
+ WHERE lhs.materialization_id IS NULL OR rhs.materialization_id IS NULL;
+ continuous_aggregate |      to_timestamp      |      to_timestamp      
+----------------------+------------------------+------------------------
+ tstz_temperature_15m | 2025-04-25 11:00:00+02 | 2025-04-26 11:15:00+02
+(1 row)
+
+SELECT materialization_id,
+       to_timestamp(lowest_modified_value),
+       to_timestamp(greatest_modified_value)
+  FROM get_raw_materialization_ranges('timestamptz');
+ materialization_id |      to_timestamp      |      to_timestamp      
+--------------------+------------------------+------------------------
+                  8 | 2025-04-25 11:00:00+02 | 2025-04-26 11:15:00+02
+(1 row)
+
+SELECT *
+  INTO saved
+  FROM _timescaledb_functions.get_materialization_invalidations(
+       'tstz_temperature_15m'::regclass,
+       '["2025-04-25","2025-04-26"]'::tstzrange
+  );
+SELECT invalidations FROM saved;
+                     invalidations                     
+-------------------------------------------------------
+ {["2025-04-25 11:00:00+02","2025-04-26 00:00:00+02")}
+(1 row)
+
+SELECT materialization_id,
+       to_timestamp(lowest_modified_value),
+       to_timestamp(greatest_modified_value)
+  FROM get_raw_materialization_ranges('timestamptz');
+ materialization_id |      to_timestamp      |      to_timestamp      
+--------------------+------------------------+------------------------
+                  8 | 2025-04-25 11:00:00+02 | 2025-04-26 11:15:00+02
+(1 row)
+
+SELECT token FROM saved LIMIT 1 \gset
+CALL _timescaledb_functions.accept_materialization_invalidations(
+       'tstz_temperature_15m'::regclass,
+       '["2025-04-25","2025-04-26"]'::tstzrange,
+       :'token'::text
```

Comment:
Same here w.r.t. argument casts... is it needed? It seems awkward in that case.

Reply:
Which one? The `tstzrange` is needed for overload reasons, the other one is just added for consistency. If you feel strongly about it, I can remove it.